### PR TITLE
Dev-Task - removed excessive traceback error printing - Packs C

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.yml [hunk 1]
```diff
@@ -302,7 +294,9 @@ script:
...
     description: Performs a memory dump operation on the remote machine.
     execution: true
-    outputs:
-  dockerimage: demisto/carbon-black-cloud:1.0.0.26432
+    outputs: []
+  dockerimage: demisto/carbon-black-cloud:1.0.0.28782
   runonce: false
   subtype: python3
+tests:
+- No tests (auto formatted)
```
Comment (dantavori) [inline Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.yml:302]: please check

File: Packs/CortexXDR/Scripts/CortexXDRRemediationActionsWidget/CortexXDRRemediationActionsWidget.yml [hunk 2]
```diff
@@ -11,5 +11,7 @@ tags:
 timeout: '0'
 type: python
 subtype: python3
-dockerimage: demisto/python3:3.9.7.24076
+dockerimage: demisto/python3:3.10.4.30607
 fromversion: 6.0.0
+tests:
+- No tests (auto formatted)
```
Comment (dantavori) [inline Packs/CortexXDR/Scripts/CortexXDRRemediationActionsWidget/CortexXDRRemediationActionsWidget.yml:17]: .

File: Packs/CrowdStrikeFalconSandbox/Integrations/CrowdStrikeFalconSandboxV2/CrowdStrikeFalconSandboxV2.yml [hunk 3]
```diff
@@ -6,2714 +6,2633 @@ display: CrowdStrike Falcon Sandbox v2
...
-    display: Trust any certificate (not secure)
-    required: false
-    type: 8
-    additionalinfo: null
-  - name: proxy
-    display: Use system proxy settings
-    required: false
-    type: 8
-    additionalinfo: null
+- name: credentials
```
Comment (dantavori) [inline Packs/CrowdStrikeFalconSandbox/Integrations/CrowdStrikeFalconSandboxV2/CrowdStrikeFalconSandboxV2.yml:9]: too many unrelated changes here, can we avoid that?

**Review by dantavori — APPROVED** [review]

**Comment by MLainer1** [comment]
This pr needs force merge due to validation errors that are unrelated to the current issue
image
RM109 - Missing README
CJ105 - No test playbook
